deny.toml

```diff
@@ -0,0 +1,37 @@
+# This section is considered when running `cargo deny check licenses`
+# More documentation for the licenses section can be found here:
+# https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html
+[licenses]
+# The lint level for crates which do not have a detectable license
+unlicensed = "deny"
+# List of explictly allowed licenses
+# See https://spdx.org/licenses/ for list of possible licenses
+# [possible values: any SPDX 3.7 short identifier (+ optional exception)].
+allow = [
+    "MIT",
+    "Apache-2.0",
+]
+# Lint level for licenses considered copyleft
+copyleft = "warn"
```

Comment:
We already have a warning from this. But warnings get buried inside console output and don't show at the top. I suspect our configuration should be binary: all licenses except for explicitly approved licenses result in an error.